test/test_basic.py

```diff
@@ -51,6 +51,7 @@ def make_simple_cohort(merge_type="union", **kwargs):
 
     return Cohort(
         patients=patients,
+        ensembl_version=75,
```

Comment:
See comment https://github.com/hammerlab/cohorts/pull/98/files#r72807318; I think it's cleaner to keep the tests the same and not require this param when Kallisto isn't relevant?